homeassistant/components/smarttub/__init__.py

```diff
@@ -0,0 +1,54 @@
+"""SmartTub integration."""
+import asyncio
+import logging
+
+from .const import DOMAIN, SMARTTUB_CONTROLLER
+from .controller import SmartTubController
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = ["climate"]
+
+
+async def async_setup(hass, _config):
```

Comment:
`config` is our standard parameter name.

Reply:
Started a branch with these fixes